app/services/plan_service/api/plans.rb

```diff
@@ -37,7 +37,7 @@ def create(community_id:, plan:)
     # All plans should come from the external plan service and that's
     # why this function is deprecated
     #
-    def create_initial_trial(community_id:, plan:)
+    def create_initial_trial(community_id:, plan: {})
```

Comment:
Hmh... this seems to be deprecated function. Is this used anywhere? I'm thinking, can we just remove it?